src/main/scala/uk/gov/hmrc/play/frontend/bootstrap/FrontendGlobal.scala

```diff
@@ -63,13 +67,14 @@ abstract class DefaultFrontendGlobal
 
   lazy val appName = Play.current.configuration.getString("appName").getOrElse("APP NAME NOT SET")
 
+  override def deviceIdFilter = DeviceIdCookieFilter(appName, auditConnector)
```

Comment:
services should provide this wiring, thes library should not know where the config lives and how it's accessed